canonical/canonical.com
# WD-13515 Dev Create /openstack/architecture

## Done

Created /openstack/architecture
/openstack page is yet to be made
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Run through checks for new pages.
- [Design](https://www.figma.com/design/vH19y2J2YqwUfksebSUvTW/OpenStack?node-id=234-4573&node-type=frame&t=iZGNMN0v8P2t4yxm-0)
- [Copydoc](https://docs.google.com/document/d/1i8rjvhw3N2gDA5NXuKEPcftXY95zePjsURmQu4dSgGo/edit)
- [Demo](https://canonical-com-1369.demos.haus/openstack/architecture)

## Issue / Card

Fixes # [WD-13515](https://warthogs.atlassian.net/browse/WD-13512)

## Screenshots

[if relevant, include a screenshot]


[WD-13515]: https://warthogs.atlassian.net/browse/WD-13515?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
